File: ScoreSystemServer/ScoreSystemServer/src/main/java/com/score/system/entity/school/ExamClassSubjectStat.java
package com.score.system.entity.school;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.score.system.entity.BaseEntity;
import lombok.Data;

@Data
@TableName("exam_class_subject_stat")
public class ExamClassSubjectStat extends BaseEntity {
    @TableId(type = IdType.AUTO)
    private Long id;
    @TableField("exam_id")
    private Long examId;
    @TableField("course_id")
    private Long courseId;
    @TableField("class_id")
    private Integer classId;
    @TableField(exist = false)
    private String className;
    @TableField("university_level")
    private Integer universityLevel;
    @TableField("avg_score")
    private Double avgScore;
    @TableField("synergy_rate")
    private Double synergyRate;
    @TableField("synergy_count")
    private Integer synergyCount;
    @TableField("contribution_rate")
    private Double contributionRate;
    @TableField("contribution_count")
    private Integer contributionCount;
}

File: ScoreSystem/ScoreSystem/ScoreSystem/Model/ExamClassSubjectStat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScoreSystem.Model
{
    public class ExamClassSubjectStat
    {
        private long id;
        private long examId;
        private long courseId;
        private int classId;
        private string className;
        private int universityLevel;
        private double avgScore;
        private double synergyRate;
        private int synergyCount;
        private double contributionRate;
        private int contributionCount;

        public long Id { get => id; set => id = value; }
        public long ExamId { get => examId; set => examId = value; }
        public long CourseId { get => courseId; set => courseId = value; }
        public int ClassId { get => classId; set => classId = value; }
        public int UniversityLevel { get => universityLevel; set => universityLevel = value; }
        public double AvgScore { get => avgScore; set => avgScore = value; }
        public double SynergyRate { get => synergyRate; set => synergyRate = value; }
        public int SynergyCount { get => synergyCount; set => synergyCount = value; }
        public double ContributionRate { get => contributionRate; set => contributionRate = value; }
        public int ContributionCount { get => contributionCount; set => contributionCount = value; }
        public string ClassName { get => className; set => className = value; }
    }
}
